styles/app.css

```diff
@@ -0,0 +1,70 @@
+
+* {
+  box-sizing: border-box;
+}
+
+ *::before. *::after {
+   box-sizing: inherit;
+ }
+
+body {
+margin: 0;
+  }
+
+.grid {
+margin: .5rem;
+padding: 1rem;
+border: 1px solid #D2D2D2;
+  }
+
+.column1, .column2, .column3 {
+width: 32%;
+padding-left: 0.6rem;
+border: 1px solid #D2D2D2;
+border-radius: 3px;
+}
+
+.column1 {
+float: left;
+}
+
+.column2 {
+display: inline-block;
+margin-left: 1rem;
+}
+
+.column3 {
+float: right;
+}
+
+img {
+height: 26rem;
+width: auto;
+}
+
+p img {
```

Comment:
Try not to use raw element tags as selectors for anything other than base style resets.

Instead add a class to these elements in your HTML and then use those classes here in your CSS.